src/cli/options.rs

```diff
@@ -1418,20 +1417,20 @@ impl Options {
             let updates_str = unsafe { from_utf8_unchecked(&bytes) };
             RoaDefinitionUpdates::from_str(updates_str)?
         } else {
-            let mut added = HashSet::new();
-            let mut removed = HashSet::new();
+            let mut added = vec![];
```

Comment:
I see that this change was made as part of the "Better json for maps (use expliciit key) and add deterministic ordering (make it testable)". I assume therefore that the issue addressed here was that the order of iteration over the map entries was non-deterministic, but if so this change also presumably permits `krillc` to submit multiple of the same "add" or "remove" instruction whereas before the map would only store one add or remove of the same ROA definition. Is this intended effect of this change?